### PR TITLE
Update docs and version bump

### PR DESCRIPTION
**Pull request by orta**
* Highlights `danger pr`
* Explains how to do the token dance
* Fixes links for the DSL shapes
* Changes the DSL shape ordering so it's easier to read

### CODE REVIEW

**Comment by orta** [comment]
Alright :1st_place_medal:

File: source/dsl/GitDSL.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import { GitCommit } from "./Commit"
 
+// This is `danger.github`
```
Comment (tivac) [inline source/dsl/GitDSL.ts]: Should that be `danger.git`?

Comment (orta) [inline source/dsl/GitDSL.ts]: damnnnnnnnn

Comment (orta) [inline source/dsl/GitDSL.ts]: :)

**Pull request merged**